api/v1alpha1/openapi.yaml

```diff
@@ -2426,6 +2470,22 @@ paths:
                 $ref: '#/components/schemas/Error'
 components:
   schemas:
+    DeviceDecommission:
+      type: object
+      properties:
+        decommissionType:
+          type: string
+          enum:
+            - 'DefaultUnenroll'
+          description: 'Specifies the desired decommissioning state of the device'
+        reason:
+          type: string
+          description: 'Reason for decommissioning device'
+        requestedBy:
```

Comment:
What do we expect users to put here?  I would think that we'd implement an audit mechanism at some point to log each action along with the user who performed it according to their credentials.

Reply:
I believe this would be filled in by the system, but maybe this isn't the correct way to specify it then.